src/modules/PowerOCR/PowerOCR/OCROverlay.xaml

```diff
@@ -96,83 +100,67 @@
             Padding="4,8,12,8"
             HorizontalAlignment="Center"
             VerticalAlignment="Top"
-            d:Background="White"
             d:Visibility="Visible"
             Background="{DynamicResource ApplicationBackgroundBrush}"
             CornerRadius="8"
             Visibility="Collapsed">
             <Border.Effect>
                 <DropShadowEffect
                     BlurRadius="32"
-                    Direction="-90"
-                    Opacity="0.6"
-                    RenderingBias="Performance" />
+                    Opacity="0.28"
+                    RenderingBias="Performance"
+                    ShadowDepth="1" />
             </Border.Effect>
+
             <StackPanel
-                Margin="2,0"
                 HorizontalAlignment="Center"
                 VerticalAlignment="Top"
                 Orientation="Horizontal">
                 <ComboBox
                     x:Name="LanguagesComboBox"
-                    Margin="2,0"
-                    Padding="4,2,0,2"
-                    Foreground="{DynamicResource TextFillColorPrimaryBrush}"
+                    Height="32"
+                    Margin="4,0"
+                    AutomationProperties.Name="{x:Static p:Resources.SelectedLang}"
                     SelectionChanged="LanguagesComboBox_SelectionChanged">
                     <ComboBox.ItemTemplate>
                         <DataTemplate>
                             <TextBlock
                                 HorizontalAlignment="Left"
                                 VerticalAlignment="Center"
-                                FontFamily="Segoe UI"
-                                Style="{StaticResource SymbolTextStyle}"
                                 Text="{Binding NativeName}" />
                         </DataTemplate>
                     </ComboBox.ItemTemplate>
                 </ComboBox>
                 <ToggleButton
                     x:Name="SingleLineToggleButton"
-                    Width="34"
-                    Height="34"
-                    Margin="2,0"
                     d:IsChecked="True"
+                    AutomationProperties.Name="{x:Static p:Resources.ResultTextSingleLine}"
                     Click="SingleLineMenuItem_Click"
                     IsChecked="{Binding IsChecked, ElementName=SingleLineMenuItem, Mode=TwoWay}"
-                    Style="{StaticResource ToggleSymbolButton}"
-                    ToolTip="(S) Make result a single line">
-                    <TextBlock Style="{StaticResource SymbolTextStyle}" Text="" />
+                    ToolTip="{x:Static p:Resources.ResultTextSingleLineShortcut}">
+                    <ui:SymbolIcon FontSize="18" Symbol="SubtractSquare24" />
```

Comment:
Were you able to test what these icons look like on Windows 10?

Reply:
Icons appear fine on Win 10. You meant these, right? 
![image](https://github.com/microsoft/PowerToys/assets/26118718/7d6f45ff-95b5-4145-bed3-47db0cf4fc34)

Reply:
Yep! Looks great!